crates/sui-protocol-config/src/lib.rs

```diff
@@ -2374,6 +2380,11 @@ impl ProtocolConfig {
                         cfg.feature_flags
                             .prepend_prologue_tx_in_consensus_commit_in_checkpoints = true;
                     }
+
+                    // Prior to this, bridge committee on testnet was initialized formed
+                    // with 7500 voting power. Since this value is only used when
```

Comment:
type: `was initialized formed with 7500 voting power` not sure what we mean here `initially formed` or let's just drop `initialized` or not sure...